pkg/errors/errors.go

```diff
@@ -18,15 +18,17 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 // THE SOFTWARE.
 
+// Package errors contains helper functions for working with YARPC errors
+// for encoding and transport implementations.
 package errors
 
 import "go.uber.org/yarpc/yarpcerrors"
 
 // WrapHandlerError is a convenience function with the following logic:
 //
-// - If err is nil, WrapHandlerError returns nil
+// - If err is nil, WrapHandlerError returns nil.
 // - If err is a YARPC error, WrapHandlerError returns err with no changes.
-// - If err is not a YARPC error, WrapHandlerError returns a new YARPC error with code
+// - If err is not a YARPC error, WrapHandlerError returns a new YARPC error with code.
 //   CodeUnknown and message err.Error(), along with service and procedure information.
```

Comment:
Optional: fix GoDoc formatting, preferably by converting this list into a paragraph. Currently it ends up looking like this:

<img width="720" alt="screen shot 2017-07-05 at 11 42 18 am" src="https://user-images.githubusercontent.com/972790/27879598-0e79df7c-6177-11e7-95c3-df0f6b608f02.png">